apps/client/src/app/userId/[id]/page.tsx

```diff
@@ -0,0 +1,17 @@
+import QRCode from "react-qr-code";
```

Comment:
Please add 'use-client', so this component is only rendered on client, not on server

Reply:
Done.